static/app/views/issueList/tagFilter.tsx

```diff
@@ -167,24 +172,38 @@ class IssueListTagFilter extends React.Component<Props, State> {
         {!tag.isInput && (
           <SelectControl
             clearable
+            aria-label={tag.key}
             placeholder="--"
             value={this.state.value}
             onChange={this.handleChangeSelect}
-            isLoading={this.state.isLoading}
+            isLoading={isLoading}
             onInputChange={this.handleChangeSelectInput}
             onFocus={this.handleOpenMenu}
-            noResultsText={
-              this.state.isLoading ? t('Loading\u2026') : t('No results found')
-            }
+            noResultsText={isLoading ? t('Loading\u2026') : t('No results found')}
             options={
               tag.predefined
                 ? tag.values &&
                   tag.values.map(value => ({
                     value,
                     label: value,
                   }))
-                : this.state.options
+                : options
             }
+            components={{
+              Option: ({
+                label,
+                ...optionProps
+              }: OptionProps<{
+                label: string;
+                value: string;
+              }>) => {
+                return (
+                  <components.Option label={label} {...optionProps}>
+                    <span data-testid={`select-control-option-${label}`}> {label}</span>
+                  </components.Option>
+                );
+              },
+            }}
```

Comment:
Is this change needed?

Reply:
We can override it :)

I will update the PR with this change

https://testing-library.com/docs/queries/bytestid/#overriding-data-testid

Reply:
@priscilawebdev you could see what's more popular and we can go with that convention

Reply:
Testing Library recommends adopting the attribute `data-testid` where possible. should we follow their recommendation?

Reply:
> following the precedent set by React Native Web which uses a testID prop to emit a data-testid attribute on the element, and we recommend you adopt that attribute where possible

Reply:
I think what's more important than what they recommend is what the community has adopted. (e.g. consider what happens in a few years when we replace RTL :P) - https://github.com/search?q=data-testid&type=code vs https://github.com/search?q=data-test-id&type=code

Reply:
hmm .. I don't see many problems with this, as I believe that replacing this attribute would not be complicated, but for now, as our codebase already uses "data-test-id", I would prefer to keep it

Reply:
I don't think there is much value for us in churning `data-test-id` at this point. Reconfiguring RTL to fit with what we have already would be the path I take. Getting folks to change over tests is going to be enough of a chore that we don't need to compound it with changing data-test-id at the same time. If in the future we need to change we can address that problem separately.